tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw

```diff
@@ -124,11 +124,15 @@
   <data name="ChooseDirectory.PlaceholderText" xml:space="preserve">
     <value>Choose a directory</value>
     <comment>Choose directory</comment>
-</data>
+  </data>
   <data name="ChooseDirectoryPromptText" xml:space="preserve">
     <value>Choose directory on Dev Drive...</value>
     <comment>Choose directory on dev drive</comment>
-  </data>  
+  </data>
+  <data name="ChosenDirectoryNotOnDevDriveErrorText" xml:space="preserve">
+    <value>Chosen folder is not in a Dev Drive location</value>
```

Comment:
I kind of like the comment wording better than this wording. Or better yet, "Selected directory must be on a Dev Drive." ?

Reply:
Figma mock had this text- [https://www.figma.com/design/AZ03I5bFPmrOpjVzFRjVn4/Dev-Home-Settings?node-id=3851-174312&t=a7E51tW4CC15BiLB-4](https://www.figma.com/design/AZ03I5bFPmrOpjVzFRjVn4/Dev-Home-Settings?node-id=3851-174312&t=a7E51tW4CC15BiLB-4)

I will take your suggestion anyway